docs/ONT_WORKFLOW_README.md

```diff
@@ -42,7 +43,13 @@ information on our collaborators, check out their websites:
 1. Generate long reads alignment metrics from the `minimap2_aligned_bam` using LongReadSum.
 1. Generate structural variant calls from the `minimap2_aligned_bam` using CuteSV.
 1. Generate structural variant calls from the `minimap2_aligned_bam` using Sniffles.
-1. Generate small variant from the `minimap2_aligned_bam` using Nanocaller_WGS.
+1. Generate structural variant calls from the `pbmm2_aligned_bam` using Sentieon LongReadSV.
```

Comment:
So, 2 different aligners are being run and you get 3 sets of SV calls:
CuteSV, Sniffles, and LongReadSV?

Reply:
Whoops this is supposed to be minimap_2.So 1 aligner and 3 sets of SV calls.